apps/src/aichat/views/AichatView.tsx

```diff
@@ -137,12 +135,6 @@ const AichatView: React.FunctionComponent<LabProps> = () => {
     }
   }, [dispatch, signInState]);
 
-  // When the level changes or if we are viewing aichat level as a different user
```

Comment:
I moved this reset behavior into `ChatWorkspace` -- it is run in the `useEffect` with the fetch to grab a user's conversation history.